src/composables/multisigAccounts.ts

```diff
@@ -107,31 +117,32 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
             gaAccountId ? sdk.balance(gaAccountId) : 0,
           ]);
 
-          const consensus = consensusResult.decodedResult as any;
+          const consensus: IMultisigConsensus = camelCaseKeysDeep(consensusResult.decodedResult);
 
-          if (consensus.tx_hash) {
-            consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+          if (consensus.txHash) {
+            consensus.txHash = Buffer.from(consensus.txHash).toString('hex');
           }
-          consensus.expiration_height = Number(consensus.expiration_height);
-          consensus.confirmations_required = Number(consensus.confirmations_required);
-          consensus.totalConfirmations = Number(consensus.confirmed_by.length);
+          consensus.expirationHeight = Number(consensus.expirationHeight);
+          consensus.confirmationsRequired = Number(consensus.confirmationsRequired);
 
           const consensusLabel = (
-            `${consensus?.confirmed_by?.length}/${consensus.confirmations_required} ${i18n.t('common.of')} ${signers.decodedResult?.length}`
+            `${consensus?.confirmedBy?.length}/${consensus.confirmationsRequired} ${i18n.t('common.of')} ${signers.decodedResult?.length}`
           );
 
-          return {
-            ...camelcaseKeysDeep(consensus),
+          const account: IMultisigAccount = {
+            ...consensus,
             ...otherMultisig,
+            consensusLabel,
+            contractId,
+            gaAccountId,
             nonce: Number(nonce.decodedResult),
             signers: signers.decodedResult,
             version: version.decodedResult,
-            consensusLabel,
-            contractId,
             balance: convertToken(balance, -MAGNITUDE),
-            address: gaAccountId,
-            multisigAccountId: gaAccountId,
+            refusedBy: [],
           };
+
+          return account;
```

Comment:
Why do you assign a value to the account variable and then return it if it's not used anywhere?

Reply:
OK, I'll use the type on the function return.